packages/ui/src/utils/addresses.ts

```diff
@@ -9,6 +10,12 @@ export const getSmoothAddressByNetwork = (network: Network) => {
     return MEV_SP_ADDRESS_PRATER;
   } else if (network == "mainnet") {
     return MEV_SP_ADDRESS_MAINNET;
+  } else if (network == "holesky") {
```

Comment:
for all networks different than mainnet or prater, this should return "null". its easier if you do 

```
export const getSmoothAddressByNetwork = (network: Network) => {
  if (network == "prater") {
    return MEV_SP_ADDRESS_PRATER;
  } else if (network == "mainnet") {
    return MEV_SP_ADDRESS_MAINNET;
  } else {
    return null;
  }
}
```

since we treat smooth address as null when outside prater or mainnet, there is no need to return error in this method anymore